src/main/java/org/dcsa/ebl/service/EquipmentService.java

```diff
@@ -1,9 +1,8 @@
 package org.dcsa.ebl.service;
 
 import org.dcsa.core.events.model.Equipment;
+import org.dcsa.core.events.model.transferobjects.ShipmentEquipmentTO;
 import org.dcsa.core.service.ExtendedBaseService;
-import org.dcsa.ebl.model.transferobjects.EquipmentTO;
-import org.dcsa.ebl.model.transferobjects.ShipmentEquipmentTO;
 import org.springframework.transaction.annotation.Propagation;
 import org.springframework.transaction.annotation.Transactional;
 import reactor.core.publisher.Mono;
```

Comment:
Is it necessary to extends the EquipmentService? You are not doing it on SealService?

Reply:
I think we can safely remove SealService/Impl from EBL. The only method it has is `findAllByShipmentEquipmentID` which is already available in the SealRepository in Event-Core. Unsure if the extend is necessary for EquipmentService since it's not being used yet.